examples/README.md

```diff
@@ -0,0 +1,12 @@
+This folder of examples consists of:
+
+1) A set of simulations that will run as is. Any of the files in these folders 
+can be run with `python <script-name>.py`, and these files are included in 
+`waveorder`'s automated testing.  
+- `/QLIPP_simulation/`
```

Comment:
Is this name supposed to be matching the directory name?

Reply:
great eye.